apps/src/music/utils/Loader.ts

```diff
@@ -108,12 +86,6 @@ export const loadProgressionStepFromSource = async (
     );
     progressionStep = response.value.level_data;
     levelCount = 1;
```

Comment:
Can the above logic also be reduced now as the only difference is levelCount = 1 if levelSource === LevelSource.LEVEL?

Reply:
Good call; this led to a bunch more tidying up.